internal/application/application.go

```diff
@@ -217,13 +212,13 @@ func Delete(ctx context.Context, cluster *kubernetes.Cluster, appRef models.AppR
 		return err
 	}
 
-	// delete pipelineruns in tekton-staging namespace
+	// delete old staging resources in namespace (helmchart.StagingNamespace)
```

Comment:
this is the second time I see `helmchart.StagingNamespace` in parentheses. Is that an accident from search and replace, or meant to highlight the variable? I'm fine with highlighting, just asking.

Reply:
It is just highlighting. How about  `--> const helmchart.StagingNamespace` instead as highlight ?

Reply:
I don't mind either way.